samples/Compute/platform-image-policy/azurepolicy.json

```diff
@@ -3,8 +3,21 @@
     "name": "platform-image-policy",
     "properties": {
         "displayName": "Only allow a certain VM platform image",
-        "description": "This policy ensures that only UbuntuServer, Canonical is allowed from the image repository",
-        "parameters": {},
+        "description": "This policy ensures that only certain VM platform images are allowed from the image repository",
+        "parameters": {
+            "imagePublisher": {
+                "type": "Microsoft.Compute/imagePublisher",
```

Comment:
All of these parameters should be "type": "array" and in plural form (i.e. imagePublisher**s**) so that multiple options can be provided for each.

The policy engine only supports the following types of parameters: string, array, and object. For now, it doesn't do further validation on the exact type of each field.